DiceRoller/shared/build.gradle.kts

```diff
@@ -53,7 +53,7 @@ kotlin {
             }
         }
         val androidMain by getting
-        val androidTest by getting
+        val androidUnitTest by getting
```

Comment:
Is this intentional? We don't have unit tests nor instrument tests, how about removing this source set altogether?

Reply:
Intentional, it moves the source set to use the new name in 1.8. There's no real purpose of having it here, true, but it at least showcases the default project structure for multiplatform.